test/corpus/classes.txt

```diff
@@ -83,119 +89,114 @@ file class A {}
   (class_declaration
     (modifier)
     name: (identifier)
-    body: (declaration_list))
+    (declaration_list))
```

Comment:
IMO the `body` field was fine, but I don't know if there was some drawback to it. Curious if you were seeing significant code size increases due to that field.

Reply:
There's a few of these removed in places that @tamasvajk added - are they used by GH Semantic?

Reply:
I think the fields are mostly used by GH CodeQL. @hvitved knows best how much we need them.

Reply:
Having field names in general certainly makes the life easier for Treesitter based QL extractors. Field names are used to generate methods in CodeQL, e.g. [this field](https://github.com/tree-sitter/tree-sitter-ruby/blob/9d86f3761bb30e8dcc81e754b81d3ce91848477e/grammar.js#L196) in the Ruby grammar gives rise to [this QL predicate](https://github.com/github/codeql/blob/main/ruby/ql/lib/codeql/ruby/ast/internal/TreeSitter.qll#L1443), which is nicer to use than something like `getChild(0)`.